### PR TITLE
Adds random parallax layers

### PR DESCRIPTION
**Pull request by ghost**
:cl: SerJanko for epic asteroid sprites and qustinnus 
 add: parallax can now sometimes have asteroids or unidentified space-gas
/:cl:

Adds a chance for one of two types of layer to spawn in the parallax.

asteroids
image

space gas (random colors)
image
image

CREDITS TO NSV13 FOR THE ASTWEROID SPRITES!! check em out here
https://github.com/DDMers/NSV13/pull/82/files

### CODE REVIEW

**Comment by Nickvr628** [comment]
Will it always be one or the other or is it possible to see both at the same time?

Also are they different across space z levels?

**Comment by Qustinnus** [comment]
one or the other and there's only 70% chance. they appear across zlevels.

**Comment by Qustinnus** [comment]
too much layers == client lag

**Comment by Kmc2000** [comment]
>:^)

**Comment by Dorsisdwarf** [comment]
It does seem a little odd to have the blue asteroids in front with the grey one behind

**Comment by Qustinnus** [comment]
tbf the lavaland thing is more intended to be a broken moon. Look at the proportions.

**Comment by Dorsisdwarf** [comment]
I meant more in a colour design aspect rather than a realism one

**Comment by Qustinnus** [comment]
These sprites are all I got sir :(

**Comment by Cyprex** [comment]
CREDITS TO NSV13 FOR THE ASTWEROID SPRITES!! check em out here
Don't you mean ASTEWOID?

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

**Comment by Potato-Masher** [comment]
Pretty. I'm down for this.

(More asteroids in background = more space dust/meteors events for the sake of REALSIM)

File: code/_onclick/hud/parallax.dm [hunk 1]
```diff
@@ -64,24 +66,25 @@
...
 			if (PARALLAX_MED)
 				C.parallax_throttle = PARALLAX_DELAY_MED
-				C.parallax_layers_max = 2
+				C.parallax_layers_max = 3
 				return TRUE
 
 			if (PARALLAX_LOW)
 				C.parallax_throttle = PARALLAX_DELAY_LOW
-				C.parallax_layers_max = 1
+				C.parallax_layers_max = 2
```
[the remark this answers is not in the document]
Comment (Qustinnus) [reply code/_onclick/hud/parallax.dm]: I wanted to add more settings but touching prefs always breaks shit

**Pull request merged**